src/rules/comment-no-empty/index.js

```diff
@@ -0,0 +1,37 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+
+export const ruleName = namespace("comment-no-empty");
+
+export const messages = utils.ruleMessages(ruleName, {
+  expected: "Empty comments are forbidden"
```

Comment:
Can you please make this match the rule from `stylelint`. So `rejected` instead of `expected` and change the error message to match.
https://github.com/stylelint/stylelint/blob/master/lib/rules/comment-no-empty/index.js#L9-L11